mtags/src/main/scala/scala/meta/internal/pc/ScriptFirstImportPosition.scala

```diff
@@ -26,95 +22,120 @@ import scala.meta._
  */
 object ScriptFirstImportPosition {
 
-  val usingDirectives: List[String] = List("// using", "//> using")
   val shebang = "#!"
-  val ammHeaders: List[String] = List("// scala", "// ammonite")
 
   private def adjustShebang(text: String): String =
     text.replaceFirst(shebang, s"//$shebang")
 
-  def ammoniteScStartOffset(text: String): Option[Int] = {
-    val it = tokenize(adjustShebang(text)).iterator
-    startMarkerOffset(it, "/*<start>*/").map { startOffset =>
-      val offset =
-        skipPrefixesOffset(ammHeaders, it, None)
-          .getOrElse(startOffset)
+  def ammoniteScStartOffset(text: String): Option[Int] =
+    scriptStartOffset(text, "/*<start>*/")
 
-      offset + 1
-    }
-  }
+  def scalaCliScStartOffset(text: String): Option[Int] =
+    scriptStartOffset(text, "/*<script>*/")
```

Comment:
Aren't both the same now?

Reply:
ScalaCLI uses `<script>` as a marker, where Ammonite uses `<start>`